<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <widget class="QPushButton" name="controlButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>234</width>
     <height>40</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Apple Color Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green; </string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textMin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>32</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>144</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="textSec">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>491</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>144</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="preset20">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>15:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset05">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>05:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset03">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>03:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset02">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>02:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset01">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>310</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>01:00</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
